force-di/main/classes/di_Injector.cls

```diff
@@ -135,18 +135,22 @@ public class di_Injector {
      **/
     private class CustomMetadataModule extends di_Module {
 
+        private string bindingObjectApiName = null;
+
         public override void configure() {
             // TODO: Support Namespace
             for(di_Binding__mdt bindingConfig :
-                [select QualifiedAPIName, DeveloperName, NamespacePrefix,
-                        Type__c, To__c, BindingObject__c, BindingObject__r.QualifiedApiName, BindingSequence__c
-                    from di_Binding__mdt]) {
+                [select QualifiedAPIName, DeveloperName, NamespacePrefix
+                      , Type__c, To__c, BindingObject__c, BindingObject__r.QualifiedApiName
+                      , BindingObjectAlternate__c, BindingSequence__c
+                   from di_Binding__mdt]) {
                 bind(bindingConfig.DeveloperName);
                 type(bindingConfig.Type__c);
-                if(String.isNotBlank(bindingConfig.BindingObject__c)) {
-                    Schema.DescribeSobjectResult[] results = Schema.describeSObjects(new String[] { bindingConfig.BindingObject__r.QualifiedApiName.toLowerCase().trim() });
+                if(String.isNotBlank(bindingConfig.BindingObject__c) || String.isNotBlank(bindingConfig.BindingObjectAlternate__c)) {
+                    bindingObjectApiName = String.isNotBlank(bindingConfig.BindingObject__c) ? bindingConfig.BindingObject__r.QualifiedApiName.toLowerCase().trim() : bindingConfig.BindingObjectAlternate__c.toLowerCase().trim();
+                    Schema.DescribeSobjectResult[] results = Schema.describeSObjects(new String[] { bindingObjectApiName });
```

Comment:
`Schema.describeSObjects` can be an expensive call; akin to SOQL within a loop.  Should this validation functionality be moved outside of the loop, where the Schema function would be called once with all candidate binding objects, and then matched to the bindings' data?